metaflow/mflog/save_logs_periodically.py

```diff
@@ -16,10 +15,12 @@ def __init__(self):
         self._thread.start()
 
     def process_message(self, msg):
-        pass
+        if msg.msg_type == MessageTypes.SHUTDOWN:
```

Comment:
What's the intention behind this change? In other words - why get rid of the explicit `shutdown` method?

Reply:
it was never called.